src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,26 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
 
+	private String _feedbackToUser;
```

Comment:
Good job with following the coding standard to put an underscore prefix for private variables